MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2088,6 +2088,9 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
             var hasTextSubs = state.SubtitleStream != null && state.SubtitleStream.IsTextSubtitleStream && state.SubtitleDeliveryMethod == SubtitleDeliveryMethod.Encode;
             var hasGraphicalSubs = state.SubtitleStream != null && !state.SubtitleStream.IsTextSubtitleStream && state.SubtitleDeliveryMethod == SubtitleDeliveryMethod.Encode;
 
+            // If double rate deinterlacing is enabled and the input framerate is 30fps or below, otherwise the output framerate will be too high for many devices
+            var doubleRateDeinterlace = (options.DeinterlaceDoubleRate && (videoStream?.RealFrameRate ?? 60) <= 30);
```

Comment:
```suggestion
            var doubleRateDeinterlace = options.DeinterlaceDoubleRate && (videoStream?.RealFrameRate ?? 60) <= 30;
```

Reply:
Good catch, these were accidentally left in after moving it out of an if block. I've corrected it. Thanks!